tests/h/auth/worker_test.py

```diff
@@ -34,8 +34,34 @@ def test_it_leaves_valid_tickets(self, db_session, factories):
         worker.delete_expired_auth_tickets()
         assert db_session.query(AuthTicket).count() == 1
 
-    @pytest.fixture
-    def celery(self, patch, db_session):
-        cel = patch('h.auth.worker.celery', autospec=False)
-        cel.request.db = db_session
-        return cel
+
+@pytest.mark.usefixtures('celery')
+class TestDeleteExpiredTokens(object):
+    def test_it_removes_expired_tokens(self, db_session, factories):
+        tokens = [
+            Token('luke', expires=datetime(2014, 5, 6, 7, 8, 9)),
```

Comment:
You've got the `factories` fixture in the params here, but aren't using it. Probably best to add a factory for tokens and use it here?